### PR TITLE
Local source scaling

### PR DESCRIPTION
**Pull request by ford-prefect**
This fixes issue #170

### CODE REVIEW

File: local/owr_local_media_source.c [hunk 1]
```diff
@@ -470,17 +518,38 @@ static GstElement *owr_local_media_source_request_source(OwrMediaSource *media_s
...
+                gst_caps_foreach(tmp, fix_video_caps_format, source_caps);
+                gst_caps_unref(tmp);
+
+                gst_caps_unref(device_caps);
+                device_caps = gst_pad_query_caps(srcpad, source_caps);
+
+                if (gst_caps_is_empty(device_caps)) {
+                    /* Accepting any format didn't work, we're going to hope that scaling fixes it */
+                    CREATE_ELEMENT(source_process, "videoscale", "video-source-scale");
+                    gst_bin_add(GST_BIN(source_pipeline), source_process);
```
Comment (sdroege) [inline local/owr_local_media_source.c]: Why so complicated? Currently we only support raw caps anyway, so just always put a videoconvert and videoscale here :smile: More work is needed in these areas for compressed support anyway

**Comment by stefanalund** [comment]
Would this help/solve: https://github.com/EricssonResearch/openwebrtc/issues/150

**Comment by sdroege** [comment]
No, this is unrelated to #150

File: local/owr_local_media_source.c (hunk 1 above)
Comment (ford-prefect) [inline local/owr_local_media_source.c]: I actually wrote the first of the two patches for my compressed work, so doing it this way is a little less work while merging in that code.

Comment (sdroege) [inline local/owr_local_media_source.c]: Ok :smile: Good to go then IMHO

**Pull request closed without merging**